src/signals.h

```diff
@@ -0,0 +1,19 @@
+#ifndef REDSHIFT_SIGNALS_H
```

Comment:
Can you add the license header to these files? These can just be copied from redshift.c and changed to the right file name/author/date.